SpatialGDK/Source/SpatialGDKServices/SpatialGDKServices.Build.cs

```diff
@@ -29,7 +31,8 @@ public SpatialGDKServices(ReadOnlyTargetRules Target) : base(Target)
 				"Json",
 				"JsonUtilities",
 				"UnrealEd",
-				"Sockets"
+				"Sockets",
+                "SpatialGDK"
```

Comment:
Format the code.

Reply:
I revert this cs file as can not directly access SpatialGDK module in SpatialGDKServices, will result in "failed to load SpatialGDKService" issue when starting editor, looks like a circular dependent problem, instead using module manager to access SpatialGDK module in SpatialGDKService